packages/lab/stories/tabstrip-next/tabstrip-next.stories.tsx

```diff
@@ -0,0 +1,223 @@
+import { useState } from "react";
+import { Story } from "@storybook/react";
+import { Button, FlexLayout } from "@salt-ds/core";
+import { TabstripNextProps, TabstripNext, TabNext } from "@salt-ds/lab";
+import { AddIcon } from "@salt-ds/icons";
+import "./tabstrip-next.stories.css";
+
+export default {
+  title: "Lab/Tabs Next/Tabstrip Next",
```

Comment:
```suggestion
  title: "Lab/Tabstrip Next/Tabstrip Next",
```

Reply:
That was intentional since I want that story nested under tabs